.jenkins/pytorch/test.sh

```diff
@@ -14,6 +14,17 @@ echo "Testing pytorch"
 if [ -n "${IN_CIRCLECI}" ]; then
   # TODO move this to docker
   pip_install unittest-xml-reporting coverage
+  # install lcov with 1.15 (gcc 9 requires lcov>1.14-2) outside of workplace, so we don't need to clean it up
+  pushd ..
+  wget https://github.com/linux-test-project/lcov/releases/download/v1.15/lcov-1.15.tar.gz
+  tar -xf lcov-1.15.tar.gz
+  pushd lcov-1.15
+  sudo make install
+  popd
+  popd
+  # symbolic link: make gcov-9 as default version of gcov, because gcc's version is gcc 9
+  sudo rm -r /usr/bin/gcov
```

Comment:
Same goes for here, if we only need this for code coverage we should probably only do this in the `coverage` block we have below